Lesson2/SecondHomework.playground/Contents.swift

```diff
@@ -0,0 +1,37 @@
+/// An object, which is a list of elements with a type integer, organized according to the LIFO principle.
+class Stack {
+    private var array: [Int] = [] {
+        didSet {
+            minimalValue = array.min()
+        }
+    }
+    private var minimalValue: Int?
+    
+    func push(element: Int) {
```

Comment:
A friendly-reminder: What is the performance of `push()`/`pop()` including the time to execute `didSet` block? I expect to see `O(1)`.

Reply:
Made changes to the Stack and StackStatistics classes